File: Examenes/ExamenPSP/2DAM_PSP-main/9 - Caja fuerte asimétrica/CajaFuerte - SIN FIRMAR MENSAJES/cliente/src/main/java/ui/screens/common/Screens.java
package ui.screens.common;

public enum Screens {


    LOGIN(ScreenConstants.FXML_LOGIN_FXML),

    PRINCIPAL(ScreenConstants.FXML_PRINCIPAL_FXML),

    REGISTER(ScreenConstants.FXML_REGISTER_FXML),

    MAIN_FOLDERS(ScreenConstants.FXML_MAIN_FOLDERS_FXML), MAIN_MESSAGES(ScreenConstants.FXML_MAIN_MESSAGES_FXML);

    private final String ruta;

    Screens(String ruta) {
        this.ruta = ruta;
    }

    public String getRuta() {
        return ruta;
    }


}

File: Examenes/ExamenPSP/2DAM_PSP-main/11 - Spring GraphQL/graphqlCliente/src/main/kotlin/ui/common/Screens.kt
package ui.common

enum class Screens(val ruta: String) {
    LOGIN("/fxml/login.fxml"), // "ui/principal/Login.fxml"
    PRINCIPAL("/fxml/principal.fxml"), // "ui/principal/Principal.fxml"
    REGISTER("/fxml/register.fxml"), // "ui/principal/Register.fxml"
    LIST_PERSONAS("/fxml/listPersonas.fxml") // "ui/principal/ListPersonas.fxml"
    ,
    MY_PROFILE("/fxml/myProfile.fxml"),
    LIST_MASCOTAS("/fxml/listMascotas.fxml")


}
//    var PRINCIPAL: Screens? = "principal.fxml"
//
//    var REGISTER: Screens? = null
//
//    var MAIN_FOLDERS: Screens? = null
//
//    var MAIN_MESSAGES: Screens? = null
//
//    private var ruta: String? = null

